services/common/src/components/forms/BaseInput.tsx

```diff
@@ -62,8 +62,8 @@ import { WrappedFieldProps, WrappedFieldMetaProps, WrappedFieldInputProps } from
 export interface BaseInputProps extends WrappedFieldProps {
   meta: WrappedFieldMetaProps;
   input: WrappedFieldInputProps;
-  label?: string;
-  labelSubtitle?: string;
+  label?: string | ReactNode;
```

Comment:
pretty much just updating this file to accept elements as well as strings for labels.